techmatters/hrm
# CHI-2974 CLI tool for republishing contacts to s3 lake

## Description

- This PR adds the cli tool for a new admin endpoint `/contacts/republish`. 
- See documentation on how to use https://techmatters.app.box.com/notes/1722659624579

### Checklist

### Other Related Issues

None

### Verification steps
- Ensure this branch is deployed to the appropriate environment
- Log into devops instance to run the 'republish'. 
  - `npm run admin-cli republish hrm -- --contacts -e <env> -r <region> -a <accountSid> -f <dateFrom>-t <dateTo>`
  - See documentation on how to use https://techmatters.app.box.com/notes/1722659624579
- To Verify, check the data lake s3 bucket to ensure the data has been updated
### AFTER YOU MERGE

1. Cut a release tag using the GitHub workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P
